contracts/test/DrawIDAndStructMappingBinarySearch.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.6;
+import "../abstract/IdBinarySearch.sol";
+
+contract DrawIDAndStructMappingBinarySearch is IdBinarySearch {
```

Comment:
This contract isn't used anywhere- let's remove it!

We *could* just have a mapping(uint32 => uint256) for DPR in a subclass.  That way you can simply inherit from PrizeTierHistory